### PR TITLE
Feature/avhrr radiance2ioda

### PR DESCRIPTION
**Pull request by HamidehGMAO**
## Description

this is a ioda-converter to convert the AVHRR radiance data to ioda-v2 fprmat. there are several sources of data for AVHRR, this converter is for patmosx data in nc format "https://www.ncei.noaa.gov/data/avhrr-hirs-reflectance-and-cloud-properties-patmosx/access/"

### Issue(s) addressed

- fixes #<730>

### CODE REVIEW

**Review by ShastriPaturi — APPROVED** [review]
Ctest passes.
Looks good.

File: src/marine/avhrr_radiance2ioda.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2019-2021 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+from __future__ import print_function
```
Comment (srherbener) [inline src/marine/avhrr_radiance2ioda.py:10]: This isn't required anymore. It was originally put in to be able to use python3 print syntax while running python2.

File: src/marine/avhrr_radiance2ioda.py [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    # (needed because we are bypassing ExtractObsData within BuildNetcdf)
+    VarDims = {
+        'brightness_temperature': ['nlocs', 'nchans']
+    }
+
+    nchans = len(chan_number)
+    nlocs = len(obs_data[('longitude', 'MetaData')])
+    DimDict = {
+        'nlocs': nlocs,
+        'nchans': nchans
```
Comment (srherbener) [inline src/marine/avhrr_radiance2ioda.py:270]: This is resulting in the nchans variable being set to all zeros, where it should be set to the list of channel numbers. The API will accept a list of the channel numbers which will then assign those channel numbers to the nchans variable. You should be able to fix this by changing this line to:

`'nchans': list(chan_number)`

**Review by srherbener — CHANGES_REQUESTED** [review]
Thanks for contributing this! Everything looks great except that the nchans variable is getting set to all zeros. Please see the comment for a recommended fix. Thanks!

File: src/marine/avhrr_radiance2ioda.py (hunk 1 above)
Comment (HamidehGMAO) [reply src/marine/avhrr_radiance2ioda.py:10]: Ah I see, Thanks for the information. I'll delete it

File: src/marine/avhrr_radiance2ioda.py (hunk 2 above)
Comment (HamidehGMAO) [reply src/marine/avhrr_radiance2ioda.py:270]: I see. Thanks

**Review by srherbener — APPROVED** [review]
thanks for addressing my comments! Sorry about the delay getting back to this.

**Pull request merged**